backend/src/config/index.ts

```diff
@@ -35,32 +36,33 @@ const required = {
 }
 
 const server = {
-  CLIENT_URI: env.CLIENT_URI || 'http://localhost:3000',
-  GRAPHQL_URI: env.GRAPHQL_URI || 'http://localhost:4000',
-  JWT_EXPIRES: env.JWT_EXPIRES || '2y',
+  CLIENT_URI: env.CLIENT_URI ?? 'http://localhost:3000',
+  GRAPHQL_URI: env.GRAPHQL_URI ?? 'http://localhost:4000',
+  JWT_EXPIRES: env.JWT_EXPIRES ?? '2y',
 }
 
 const hasDKIMData = env.SMTP_DKIM_DOMAINNAME && env.SMTP_DKIM_KEYSELECTOR && env.SMTP_DKIM_PRIVATKEY
```

Comment:
this is no longer needed?!

Reply:
Oh, good catch, I will add this.

Typescript is not smart enough to understand what we're doing. The solution is to implement a type-guard for `config` objects (I was too lazy for that) or just enumerate the respective local variables.

Reply:
Ich benutze `ts-dotenv` hier um die types sauber zu definieren https://github.com/IT4Change/reformer.network/blob/master/badgeLookupClient/src/env.ts

Reply:
Cool! Genau so etwas suche ich oft. Aber das NPM package scheint seit Jahren nicht mehr gewartet worden zu sein.